.about.yml

```diff
@@ -3,24 +3,24 @@
 
 # This is a short name of your project that can be used as a URL slug.
 # (required)
-name: atf-eregs
+name: omb-eregs
 
 # This is the display name of your project. (required)
-full_name: ATF eRegulations
+full_name: OMB eRegulations
 
 # What is the problem your project solves? What is the solution? (required)
-description: The Bureau of Alcohol, Tobacco, Firearms and Explosives (ATF) maintains a large amount of complex information that the public needs to use, including regulations with important requirements related to munitions, firearms, and explosives. 18F adapted and customized eRegulations, an open source project started by the Consumer Financial Protection Bureau in 2013, to make ATF's regulations easier to find, read, and understand.
+description: The Office of Management and Budget (OMB) has policy documents they would like to be in a more usable and reference-able form. We are implementing an instance of eRegulations and expanding the platform to support additional document types.
```

Comment:
I think we'll probably want to swap `policy documents` for `policies` based on what Matt's saying, but we can get to that later.